ansible/roles-infra/infra-ibm-ssh-key/tasks/main.yml

```diff
@@ -13,11 +13,12 @@
 #     | cut -d " " -f2
 #   register: r_ssh_key_fingerprint
 
+# If we only need one key on the sandbox, we can use the task below to do both, check and create
+# Idempotency tested
 - name: Create SSH key if it does not exist
   when: (r_ssh_keys.stdout | from_json) | json_query(ssh_key_query) | length == 0
   command: >-
-    ibmcloud is keyc {{ guid }}-key "{{ infra_ibm_ssh_key_pubssh_key }}"
-    --json --resource-group-id {{ resource_group_id }}
+    ibmcloud is key-create {{ guid }}-key "{{ infra_ibm_ssh_key_pubssh_key }} --resource-group-name {{ guid }}-rg"
```

Comment:
I'm going to keep this with ID for now since we have it from the previous RG create and/or retrieve.

Reply:
Agreed, commit will follow.